src/main/kotlin/com/sourcegraph/cody/sidebar/VSCodeInjected.kt

```diff
@@ -0,0 +1,40 @@
+package com.sourcegraph.cody.sidebar
+
+// #_vscodeApiScript
+val VSCODE_INJECTED_API_SCRIPT =
```

Comment:
This is unused.

Reply:
I am going to remove it. @dominiccooney - let me know if it's needed and what for

Reply:
Yeah, this is some detritus from the early bring up. Thanks for removing it.